nvflare/job_config/fed_job_config.py

```diff
@@ -148,12 +148,14 @@ def simulator_run(self, workspace, clients=None, n_clients=None, threads=None, g
                     + workspace
                 )
                 if clients:
+                    clients = self._trim_whitespace(clients)
```

Comment:
Since this is critical to our Job API, 
can you add some test cases in the unit test to show how effectice/correct is this method?

Reply:
added unit test.